src/MainGrid.js

```diff
@@ -411,6 +411,28 @@ MainGrid.prototype.defineCrosshairBehaviour = function () {
             if (eventType === 'mousemove' && typeof _self.selectionRegion !== 'undefined') {
                 _self.changeSelection(coord);
             }
+
+            var xIndex = _self.width < coord[0] ? -1 : _self.rangeToDomain(_self.x, coord[0]);
+            var yIndex = _self.height < coord[1] ? -1 : _self.rangeToDomain(_self.y, coord[1]);
+
+            var donor = _self.donors[xIndex];
+            var gene = _self.genes[yIndex];
+            var obsArray = _self.nullableObsLookup(donor, gene);
+
+            var html = Mustache.render(_self.templates.mainGridCrosshair, {
+                donor: donor,
+                gene: gene,
+                obs: obsArray
+            });
+
+            if(html) {
+                _self.div.transition()
+                    .duration(200)
+                    .style('opacity', 0.9);
+                _self.div.html(html)
+                    .style('left', (d3.event.pageX + 15) + 'px')
+                    .style('top', (d3.event.pageY + 30) + 'px');
```

Comment:
Could we change these to use a transform(and in the not crosshair tooltip as well)? If a parent container is position absolute or relative the tooltip could be to far from the mouse